barretenberg/cpp/src/barretenberg/api/api_client_ivc.cpp

```diff
@@ -210,6 +212,46 @@ void ClientIVCAPI::write_solidity_verifier([[maybe_unused]] const Flags& flags,
     throw_or_abort("API function contract not implemented");
 }
 
+bool ClientIVCAPI::check_ivc(const std::filesystem::path& input_path)
```

Comment:
The word "check" is kind of loaded (based on the "check circuit" lingo) and this is something distinctly different. Can you name this something more descriptive like `check_precomputed_vks` or something?